web/app.fluidity.money/app/routes/wtf.tsx

```diff
@@ -296,7 +296,7 @@ export default function IndexPage() {
               accessors={{
                 xAccessor: (d: HighestRewardMonthly & { x: number }) => d.x,
                 yAccessor: (d: HighestRewardMonthly) =>
-                  Math.log(d.winning_amount_scaled + 1),
+                  Math.log(d.winning_amount + 1),
```

Comment:
Instead of this, can you use yAccessor = d.winning_amount_scaled * 100000 (Get rid of Math.log). Reason being tokens may have different decimal places